### PR TITLE
test: verify kokoro pull request

### PR DESCRIPTION
**Pull request by mpeddada1**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/java-texttospeech/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> ☕️

If you write sample code, please follow the [samples format](
https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/SAMPLE_FORMAT.md).

### CODE REVIEW

**Comment by mpeddada1** [comment]
Printing the current branch returns `pull_branch` instead of `check-kokoro`. Selecting when to run a particular test in kokoro based on the branch name doesn't seem as straightforward atm:

```
Current branch: pull_branch
Another current branch: pull_branch
  main
* pull_branch
```

File: .kokoro/build.sh [hunk 1]
```diff
@@ -71,8 +71,29 @@ integration)
...
+#      ;;
+#    *)
+#      # For now, only generate documentation on pushes to `master`.
+#      echo "The branch name is NOT check-kokoro."
+#      exit 0
+#      ;;
+#    esac
+#
+    CURRENT_BRANCH=$(git branch | grep \* | cut -d ' ' -f2)
+    echo "Current branch: ${CURRENT_BRANCH}"
```
Comment (suztomo) [inline .kokoro/build.sh:93]: Can you add "printenv" command to print all environment variables? There might be a variable that holds what you're looking for.

Comment (mpeddada1) [reply .kokoro/build.sh:95]: great idea! Just tried this out and unfortunately there doesn't seem to be a variable for the kokoro branch name but doing these two commands looks promising:

```
branchName=$(git name-rev 9188d10f710ce97b884dc8c805002ded08144e70)
echo $branchName
```

Result: `9188d10f710ce97b884dc8c805002ded08144e70 remotes/origin/check-kokoro~24`

The `KOKORO_GIT_COMMIT` could be useful.

**Comment by mpeddada1** [comment]
Working solution:

```
    branchName=$(git name-rev $KOKORO_GIT_COMMIT | sed 's/.* //')
    if [[ "${branchName}" == *"check-kokoro"* ]]; then
       # Run Unit and Integration Tests with Native Image
      mvn -B ${INTEGRATION_TEST_ARGS} -ntp -Pnative -Penable-integration-tests test
      RETURN_CODE=$?
    fi
```

**Pull request closed without merging**